utils/subprocess/executor_test.go

```diff
@@ -431,3 +436,47 @@ func TestCancelledSubprocess3(t *testing.T) {
 		})
 	}
 }
+
+func TestOutputWithEnvironment(t *testing.T) {
+	if platform.IsWindows() {
+		t.Skip("access denied")
+	}
+	defer goleak.VerifyNone(t)
+	logger, err := logs.NewLogrLogger(logstest.NewTestLogger(t), "test")
+	require.NoError(t, err)
+	t.Run("happy", func(t *testing.T) {
+		output, err := OutputWithEnvironment(context.Background(), logger, nil, "du", "-h")
+		require.NoError(t, err)
+		assert.NotEmpty(t, output)
+		fmt.Println(output)
+	})
+	t.Run("happy with output", func(t *testing.T) {
+		testString := fmt.Sprintf("This is a test %v!", faker.Sentence())
```

Comment:
You night want to wrap these in `'` just in case `faker.Sentence()` creates some valid shell code (as unlikely as that is...)
```suggestion
		testString := fmt.Sprintf("'This is a test %v!'", faker.Sentence())
```

Same for the other ones